src/test/java/marquez/db/mappers/DbTableVersionRowMapperTest.java

```diff
@@ -40,7 +40,10 @@
 
   @Test
   public void testMap() throws SQLException {
+    final Object exists = mock(Object.class);
     final ResultSet results = mock(ResultSet.class);
+    when(results.getObject(Columns.CREATED_AT)).thenReturn(exists);
```

Comment:
How does this reconcile with line 48?
```
when(results.getTimestamp(Columns.CREATED_AT)).thenReturn(Timestamp.from(CREATED_AT));
```

Reply:
Sure, see [`Columns.timestampOrThrow()`](https://github.com/MarquezProject/marquez/blob/master/src/main/java/marquez/db/Columns.java#L93)